system_test/aest_hard_fork_SUITE.erl

```diff
@@ -755,14 +751,22 @@ to_forms(DotEndingTokenLists) ->
     lists:map(fun(Ts) -> {ok, F} = erl_parse:parse_form(Ts), F end,
               DotEndingTokenLists).
 
-run_erl_cmd_on_node(NodeName, ErlCmd, ExpectedOutput, Timeout, Cfg) ->
-    ct:log("Running Erlang command on node ~s:~n~s~nExpecting: ~s",
-           [NodeName, ErlCmd, ExpectedOutput]),
+make_erl_cmd(Fmt, Params) ->
+    lists:flatten(io_lib:format(Fmt, Params)).
+
+run_erl_cmd_on_node(NodeName, ErlCmd, Timeout, Cfg) ->
+    ct:log("Running Erlang command on node ~s:~n~s", [NodeName, ErlCmd]),
     Cmd = ["bin/epoch", "eval", ErlCmd],
     {0, Output} = aest_nodes:run_cmd_in_node_dir(NodeName, Cmd, Timeout, Cfg),
-    ?assertEqual(ExpectedOutput, string:trim(Output)),
-    ct:log("Run Erlang command on node ~s", [NodeName]),
-    ok.
+    Result = eval_expression(Output),
```

Comment:
Let us hope we don't accidentally have "file:delete_all()" as output of the docker...